roles/repositories/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: 'Enable Repositories'
+  theforeman.foreman.repository_set:
+    username: "{{ username }}"
+    password: "{{ password }}"
+    server_url: "{{ server_url }}"
+    validate_certs: "{{ validate_certs }}"
+    organization: "{{ organization }}"
+    product: "{{ item.0.name }}"
+    name: "{{ item.1.name }}"
+    state: enabled
+    repositories:
```

Comment:
What if I want to enable multiple repo-sets of the same product? I think currently this will fail (or rather the last repo-set will win)

Reply:
This has not been the case in my testing. For example the test playbook here has 2 repository sets under the same product, and both are successfully enabled.

I admit I am very impressed by Ansible's `with_subelements`

Reply:
Cool,thanks!